src/vivarium_public_health/results/mortality.py

```diff
@@ -114,22 +114,20 @@ def register_observations(self, builder: Builder) -> None:
                 self.causes_to_stratify,
                 requires_columns=["cause_of_death"],
             )
-        builder.results.register_adding_observation(
+        self.register_adding_observation(
+            builder=builder,
             name="deaths",
             pop_filter=pop_filter,
-            when="collect_metrics",
```

Comment:
This is always happening on collect metrics now right?

Reply:
Correct, that's the default (so I didn't precisely need to remove it but it seems like we typically don't include)

Reply:
Oh it is set below. Silly file order